examples/3d-tiles/package.json

```diff
@@ -4,6 +4,7 @@
     "start": "webpack-dev-server --progress --hot --open",
     "start-local": "webpack-dev-server --env.local --progress --hot --open",
     "start-local-deck": "webpack-dev-server --env.local --env.deck --progress --hot --open",
+    "start-local-luma": "webpack-dev-server --env.local --env.luma --progress --hot --open",
```

Comment:
Is this config `env.luma` handleed in webpack config file?

Reply:
> Is this config env.luma handleed in webpack config file?

Will add in follow-up, missed it because too many PRs in play...